src/idl_gen_python.cpp

```diff
@@ -61,11 +100,15 @@ static void BeginEnum(const std::string class_name, std::string *code_ptr) {
   code += "class " + class_name + "(object):\n";
 }
 
+std::string EscapeKeyword(const std::string &name) {
+  return Keywords.find(name) == Keywords.end() ? name : name + "_";
+}
+
 // A single enum member.
 static void EnumMember(const EnumVal ev, std::string *code_ptr) {
   std::string &code = *code_ptr;
   code += Indent;
-  code += ev.name;
+  code += EscapeKeyword(ev.name);
```

Comment:
This function should ideally be called everywhere a `.name` gets put in generated code, not just here.